pkg/resource/api_test.go

```diff
@@ -250,17 +250,20 @@ func TestAPIPatchingApplicator(t *testing.T) {
 		args   args
 		want   want
 	}{
-		"NotAMetadataObject": {
-			reason: "An error should be returned if we can't access the object's metadata",
-			c:      &test.MockClient{MockGet: test.NewMockGetFn(errBoom)},
-			args: args{
-				o: &nopeject{},
-			},
-			want: want{
-				o:   &nopeject{},
-				err: errors.New("cannot access object metadata"),
+		// TODO: fix this crashing test
```

Comment:
I have to find out why this test segfaults. :thinking:

Reply:
Yeah that's true, the question is if we still need those checks or if we can remove the test case…